scripts/e2e/run-e2e.sh

```diff
@@ -36,4 +36,4 @@ InstallAGIC
 
 # run test
 go mod init
-go test -v --tags e2e ./... > testoutput.txt || { echo "go test returned non-zero"; cat testoutput.txt; exit 1; }
+go test -v -timeout 60m -tags e2e ./... > testoutput.txt || { echo "go test returned non-zero"; cat testoutput.txt; exit 1; }
```

Comment:
why 60mins here? maybe we should set timeout per test case?

Reply:
lets tackle that seperately.